docs/developer-docs/integrations/sns/managing/sns-asset-canister.md

```diff
@@ -19,15 +19,14 @@ The term **update** refers to changing or updating the assets stored within an a
 
 ## Deploying an asset canister
 
-An asset canister must first be deployed before control of it can be handed over to an SNS. When a dapp's control is handed over to an SNS, this is also true for the associated asset canister.
-
 The general overview of deploying an asset canister during an SNS launch is as follows:
-- First, the asset canister must be created with or upgraded to a Wasm file from [dfx 0.14.1+](https://github.com/dfinity/sdk/blob/release-0.14.1/src/distributed/assetstorage.wasm.gz).
-- Then, the dapp should hand control of the asset canister over to the SNS by setting the following permissions:
-    - The SNS governance canister is given `Commit` permissions. This is done by the previous developer using the `grant_permissions` command (see the granting permissions section below), otherwise the SNS must grant this permission once it is a controller.
-    - A whitelist of principals with `Prepare` permissions is created to give certain individuals the permission to upload changes to the asset canister. Changes must be approved through a proposal before they are applied to the asset canister. 
-    - The user or developer creating the SNS should remove their own personal permissions. 
-- Lastly, the SNS's function should be registered to commit the configuration.
+- First, the asset canister must be created with or upgraded to a Wasm file from [dfx 0.15.2+](https://github.com/dfinity/sdk/blob/release-0.15.2/src/distributed/assetstorage.wasm.gz).
+- Then, developers should use `revoke_permission` to remove their own permissions (especially `Commit`) that allow them to update the assets arbitrarily.
+- Then, after launching the SNS, the SNS's function should be registered to commit proposed changes.
+- Last, using upgrade arguments, updated permissions can be set. This is only possible after launching an SNS because before the launch the principal id of SNS governance is not known yet
```

Comment:
This would would be possible during the SNS swap already, right?
So maybe saying that this is only possible after the SNS canisters have been deployed would be more precise?